hillel/locking/locking.js

```diff
@@ -0,0 +1,25 @@
+/*
+Написати функцію, яка приймає 1 параметр. з тим, що передали перший раз і т. д.
+ Все це із замиканнями, наприклад: sum(3) = 3 sum(5) = 8 sum(20) = 28
+*/
+
+ function outsideF(startVar){
+    in_lock=startVar;
+    return function(newVar){
+        return startVar+newVar
+    }
+
+}
```

Comment:
Я запустив Вашу функцію з прикладом із завдання. Здається Ви не зовсім зрозуміли що треба було зробити

```
const sum100500n = outsideF(0);

console.warn(sum100500n(3)); // 3 - ok
console.warn(sum100500n(5)); // 5 instead of 8
console.warn(sum100500n(20)); // 20 instead of 28

```